blockchains/erc20/erc20_worker.js

```diff
@@ -63,17 +94,23 @@ class ERC20Worker extends BaseWorker {
 
     logger.info(`Fetching transfer events for interval ${result.fromBlock}:${result.toBlock}`);
 
-    let events;
+    let events = [];
     let overwritten_events = [];
     if ('extract_exact_overwrite' === constants.CONTRACT_MODE) {
-      events = await contractEditor.getPastEventsExactContracts(this.web3, result.fromBlock, result.toBlock,
-        new TimestampsCache());
-      contractEditor.changeContractAddresses(events);
+      const timestampsCache = new TimestampsCache();
+      events = await getPastEvents(this.web3, result.fromBlock, result.toBlock, this.allOldContracts, timestampsCache);
```

Comment:
We fetch the events for all overwritten contracts at once now. Apparently the Node API supports it, so we can reduce the number of requests. Before this change we used to do one request per overwritten contract.